exporter/loggingexporter/README.md

```diff
@@ -15,7 +15,10 @@ Supported pipeline types: traces, metrics, logs
 The following settings are optional:
 
 - `loglevel` (default = `info`): the log level of the logging export
-  (debug|info|warn|error). When set to `debug`, pipeline data is verbosely
+  (debug|info|warn|error). When set to `debug`, pipeline data is verbosely 
```

Comment:
> @mx-psi I didn't find a neat way to mention loglevel is deprecated, I have added a note section to mention about the same.

What about something like this?

```suggestion
  (debug|info|warn|error). When set to `debug`, pipeline data is verbosely 
      - **Note**: This option has been deprecated in favor of `verbosity`
```

I think it's important that the note is as close as possible to the setting description